src/main/java/co/omise/models/Source.java

```diff
@@ -121,6 +131,8 @@ public void setTerminalId(String terminalId) {
         private String storeName;
         @JsonProperty("terminal_id")
         private String terminalId;
+        @JsonProperty("installment_terms")
+        private String installmentTerms;
```

Comment:
Is it should be the integer?

Reply:
The return type was a String, so I set it as String everywhere to keep it consistent.

Reply:
👌